Update dynamo refresh warning (#40)
MikeInnes/IRTools.jl: docs/src/dynamo.md

!!! warning

>     On Julia versions older than 1.3, a current usability issue with the dynamo is that it is _not_ automatically updated when you redefine functions. For example:

Reviewer: ```suggestion
    On Julia versions older than 1.3, dynamos are _not_ automatically updated when you redefine functions. For example:
```

    julia> roundtrip(foo, 5)
    6
    ```
    

>     On versions newer than 1.3, `IRTools.refresh` is not required. 

Reviewer: ```suggestion
    With Julia 1.3 and later, `IRTools.refresh` is not required. 
```
